tests/FINDOLOGIC/Api/Tests/Responses/Xml21/Xml21ResponseTest.php

```diff
@@ -289,9 +304,15 @@ public function testResponseWillReturnWeightsOfItemsAsExpected()
     {
         // Weights do have a float value, but checking the value to its 1:1 value is unnecessary.
         $expectedWeight = [
-            0.10730088502169, 0.3296460211277, 0.90265488624573, // Farbe
-            0.038716815412045, 0.63053095340729, 0.12168141454458, // Material
-            0.0022123893722892, 0.08517698943615, 0.13495574891567, // Hersteller
+            0.10730088502169,
```

Comment:
There was a reason for having certain values in the same line. Please move it back how it was before (even if that is not PSR-12 conform, this library is not yet PSR-12 ready)

Reply:
Do this for all array changes you've made here.